src/main/_docs/admin-guide/admin-configuration.md

```diff
@@ -159,14 +159,43 @@ CODENVY_MACHINE_DOCKER_PRIVILEGE_MODE=true
 If you are running a private registry internally to your company, you can [optionally mirror Docker Hub](https://docs.docker.com/registry/recipes/mirror/). Your private registry will download and cache any images that your users reference from the public Docker Hub. You need to [configure your Docker daemon to make use of mirroring](https://docs.docker.com/registry/recipes/mirror/).
 
 ### Using Docker In Workspaces
-If you'd like your users to work with projects which have their own Docker images and Docker build capabilities inside of their workspace, then you need to configure the workspace to work with Docker. You have two options:
+If you'd like your users to work with projects which have their own Docker images and Docker build capabilities inside of their workspace, then you need to configure the workspace to work with Docker. You have three options:
 
-1. Activate Docker's prvileged mode, where your user workspaces have access to the host.
-2. Configure Codenvy workspaces to volume mount their host Daemon when starting
+1. Activate Docker's privileged mode, where your user workspaces have access to the host.
+```shell
+# Update your codenvy.env to allow all Codenvy workspaces machines/containers privileged rights:
+CHE_DOCKER_PRIVILEGED=true;
+```
+2. Configure Codenvy workspaces to volume mount the host docker daemon socket file.
+```shell
+# Update your codenvy.env to allow all Codenvy workspaces to volume mount their host Daemon when starting:
+CODENVY_MACHINE_SERVER_EXTRA_VOLUME=/var/run/docker.sock:/var/run/docker.sock;
```

Comment:
This is only for using docker.sock inside a workspace. Yes for the codenvy server we always volume mount docker.sock into it but we do not volume mount into workspace machines the docker.sock file.